notebooks/ir/dense/dense_ir.ipynb

```diff
@@ -245,15 +245,34 @@
       "    \"queries\": null,\n",
       "    \"index_name\": null,\n",
```

Comment:
There might be something wrong with this file because I can not do view file it in github. I get a message saying this is not a valid notebook.

Reply:
There was a problem with the rebase - fixed now